tests/ui/useless_attribute.stderr

```diff
@@ -1,16 +1,20 @@
-error: useless lint attribute
- --> $DIR/useless_attribute.rs:5:1
+error[E0464]: multiple matching crates for `clippy_lints`
+ --> $DIR/useless_attribute.rs:9:1
   |
-5 | #[allow(dead_code, unused_extern_crates)]
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: if you just forgot a `!`, use: `#![allow(dead_code, unused_extern_crates)]`
+9 | extern crate clippy_lints;
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
-  = note: `-D useless-attribute` implied by `-D warnings`
+  = note: candidates:
+          crate `clippy_lints`: /home/mateusz/temp/rust-clippy/target/debug/deps/libclippy_lints-9e310b886f99a60d.rlib
+          crate `clippy_lints`: /home/mateusz/temp/rust-clippy/target/debug/deps/libclippy_lints-72c784bbede1859c.rlib
```

Comment:
Looks like `tests/ui/update-references.sh` misbehaved. I'm gonna cleanup this mess